WcaOnRails/app/models/competition.rb

```diff
@@ -233,6 +234,7 @@ def update_foreign_keys
       CompetitionDelegate,
       CompetitionOrganizer,
       DelegateReport,
+      CompetitionTab,
     ].each do |model|
       foreign_key = model.column_names.include?("competitionId") ? "competitionId" : "competition_id"
       model.where(foreign_key => id_was).update_all(foreign_key => id)
```

Comment:
Ok, I've just committed the code pasted above. I've also added an expectation that will fail when the callback is removed, so that we don't forget to remove the test as well =)